### DISCUSSION TITLE
What is github customer support respone time? no response after 5 days

### DISCUSSION THREAD

**Discussion by flywhc** [community/community#60468] [New to GitHub]
I have a problem with linking my account with my company account with Copilot.
I submitted a ticket 5 days ago (3 working days) but received no response til now. (ticket #2237338)

Usually, customer support should reply to something in a short time even cannot solve it, especially for a paid user.

What is the best way to contact customer support?

**Comment by mikellykels** [comment]
Same here, but it's been over 2 weeks for me. They even archived a ticket I submitted without a reason, so I have no idea if they're even addressing anything. I've followed up in my ticket an still nothing as well.

**Comment by OshekharO** [comment]
Try clearing the Copilot cache by removing the ~/.vim/copilot directory. This will force Copilot to re-authenticate and may resolve any authentication-related issues.

**Comment by queenofcorgis** [comment]
Hi there - thank you for your patience, it looks like an agent has replied to your ticket. Tickets are answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

**Reply by ghost** [reply]
@queenofcorgis , please help me. My account, @myselfshivams, has been suspended. I have opened a support ticket, but after 2-3 weeks, I have received no reply.

**Reply by queenofcorgis** [reply]
Hi @myselfshivams-temp 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

**Reply by ghost** [reply]
@queenofcorgis , I truly appreciate your response and understanding of my situation. My work is currently at a standstill, and I'm feeling quite anxious about it. I understand that you have a lot on your plate and that there are other tickets ahead of mine. However, I'm reaching out with a humble request to prioritize my ticket. Reactivating my GitHub account is crucial for me to resume my work promptly.

I assure you that I'll be immensely grateful for any expedited assistance you can provide. Your help in this matter would not only alleviate my current stress but also allow me to continue contributing to my project without further delay.
